### PR TITLE
Removing DLC Requirements from Mod

### LINKED ISSUE #880: Build Hospital quest don't require "Reaper's Due" DLC
**Mod Version**
Master branch

**What expansions do you have installed?**
No Reaper's Due

**Please explain your issue in as much detail as possible:**
Build Hospital quest don't require "Reaper's Due" DLC, so you can't complete it.

**Steps to reproduce the issue:**
Get "Build Hospital" quest

**Upload an attachment below: .zip of your save, or screenshots:**

<details>
<summary>Click to expand</summary>

image

</details>

### PR DESCRIPTION
**Pull request by MemeWizard**
**Changelog:**
- You no longer need Holy Fury for mod bloodlines (You still need Holy Fury for vanilla DLC-locked bloodlines)
- You no longer need Monks and Mystics for mod artifacts (You still need M&M for vanilla DLC-locked artifacts)
- You no longer need Jade Dragon for mod offmap OG screens and interactions
- You no longer need Jade Dragon or Horse Lords for mod tributaries (You still need Jade Dragon or Horse Lords for vanilla DLC-locked tributaries)
- Fixed #880

**Notes for Testers:**
- Turn off all DLC before starting the game
  - Test bloodline special events, like summoning great warriors

### CODE REVIEW

File: common/societies/wc_societies.txt [hunk 1]
```diff
@@ -95,10 +94,11 @@ cult_of_the_damned = {
 		}
 		decisions = {
 			the_devil_worshipers_tainted_touch
-			devil_worshipers_dark_healing
+			# devil_worshipers_dark_healing
 			ask_for_dark_gift
 			prisoner_create_monstrous_undead
 		}
+		custom_tooltip = devil_worshipers_dark_healing_power_tooltip
```
Comment (zumbak04) [inline common/societies/wc_societies.txt]: Maybe show decision, but add text like "Requires Monks and Mystics" if players don't have the DLC?

**Comment by zumbak04** [comment]
> Test bloodline special events, like summoning great warriors
See `bloodline_heroes`, `bloodline_inspire_commanders` and `bloodline_monumental` vanilla abilities. Everything work without DLCs.

File: common/societies/wc_societies.txt (hunk 1 above)
Comment (MemeWizard) [reply common/societies/wc_societies.txt]: Sounds really bad, imo.
I think that constantly reminding players after game start that they need DLC in order to use that feature it's just a bad manners. Even paradox understands that and just and hide or make inactive content that is DLC-locked.

**Comment by zumbak04** [comment]
There's a problem. The evil societies exist, but I can't hunt them.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by zumbak04** [comment]
Don't know if it's a problem, but artifacts can't be looted without Monks and Mystics, `MNM.9899` requires it.

**Comment by MemeWizard** [comment]
> Don't know if it's a problem, but artifacts can't be looted without Monks and Mystics, MNM.9899 requires it.

Just like in vanilla, you can get artifacts with Holy Furry, but can't loot them without MnM DLC

> There's a problem. The evil societies exist, but I can't hunt them.

Just like in vanilla!

<details>
<summary>Click to expand</summary>

image
image
image
image

</details>

**Comment by zumbak04** [comment]
`MNM.10030` requires Monks and Mystics, this event rank up and force AI characters to join. Without it, socieities will be pretty empty.

**Comment by MemeWizard** [comment]
Moved societies to separate branch, I will think what to do with them later.

**Comment by geffe1** [comment]
Bloodlines and such work, however- I do have one complaint, and that is about the plague- the Scourge will just sit in Northrend due to this.

Everything else appears to be fine.

**Comment by MemeWizard** [comment]
> I do have one complaint, and that is about the plague- the Scourge will just sit in Northrend due to this.

Because the Lordaeron Plague needs "The Reaper’s Due" in order to spawn and I didn't deleted "The Reaper’s Due" requirements in this PR.

So, this is work for another PR.

**Pull request merged**